cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1607,6 +1608,22 @@ runQueryRatifyState
     ratifyState <- runQuery nodeConnInfo target $ queryRatifyState eon
     writeOutput mOutFile ratifyState
 
+runQueryFuturePParams
+  :: Cmd.QueryNoArgCmdArgs era
+  -> ExceptT QueryCmdError IO ()
+runQueryFuturePParams
+  Cmd.QueryNoArgCmdArgs
+    { Cmd.eon
+    , Cmd.commons =
+      Cmd.QueryCommons
+        { Cmd.nodeConnInfo
+        , Cmd.target
+        }
+    , Cmd.mOutFile
+    } = conwayEraOnwardsConstraints eon $ do
+    futurePParams <- runQuery nodeConnInfo target $ queryFuturePParams eon
```

Comment:
Not critical for the current PR however I imagine outputting the next epoch would be useful as well.